src/Model/GalleryInterface.php

```diff
@@ -109,5 +109,10 @@ public function getGalleryHasMedias();
     /**
      * @param GalleryHasMediaInterface $galleryHasMedia
      */
-    public function addGalleryHasMedias(GalleryHasMediaInterface $galleryHasMedia);
```

Comment:
can we just remove the method without any explanation?

Reply:
I don't think it is. This interface just won't enforce classes to implement this method anymore. But it's not going to break anything.

Reply:
Sf is not ok with it: https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces

If a consumer of this interface expects a method to be there calls it, and suddenly, that method is no longer guaranteed to be there :boom: